addons/tm_tmf_loadouts/loadouts/us_army_1970_odgreen.hpp

```diff
@@ -47,45 +47,53 @@ class baseMan {// Weaponless baseclass
     // This is executed after unit init is complete. argument: _this = _unit.
     code = "";
 
-	// These are acre item radios that will be added during the ACRE init. ACRE radios added via any other system will be erased.
-	radios[] = {};
+    // These are acre item radios that will be added during the ACRE init. ACRE radios added via any other system will be erased.
+    radios[] = {};
 };
 class r : baseMan
 {
     displayName = "Rifleman (M14)";
-    headgear[] = {"rhsgref_helmet_M1_bare", "rhsgref_helmet_M1_bare_alt01"};
-	vest[] = {"usm_vest_LBE_rm"};
-    backpack[] = {"CUP_B_AlicePack_Khaki"};
+    headgear[] = {"rhsgref_helmet_M1_bare", "rhsgref_helmet_M1_bare_alt01","rhsgref_helmet_M1_liner","rhsgref_helmet_M1_painted","rhsgref_helmet_M1_painted_alt01"};
```

Comment:
I think the painted helmet variant would be out of place - can't find any evidence the US would paint their helmets in tha tstyle. The liner one is nice to add though